pkgs/by-name/op/open-vm-tools/package.nix

```diff
@@ -136,6 +136,10 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   postInstall = ''
+    # Fix https://github.com/NixOS/nixpkgs/issues/365587
+    mkdir -p $out/lib
+    ln -s ${pam}/lib/libpam.so.0 $out/lib/libpam.so.0
```

Comment:
I don't like including `libpam.so.0` in this package. An alternative might be to set the `rpath` explicitly. Alternatively maybe we need to patch the makefile or use `wrapProgram`?

Reply:
Something like this : 

```
preFixup = ''
  patchelf --set-rpath "${pam}/lib:$out/lib:$(patchelf --print-rpath $out/bin/vmtoolsd)" $out/bin/vmtoolsd
'';
```